### PR TITLE
Completed css-layout-classes.

### PR DESCRIPTION
**Pull request by saraysc**
image

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-third {
+  flex-basis: calc(200% / 3);
+}
+
+.column-one-fourth {
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:24]: make sure you are setting the proper flex basis for this element. And this class should not have display flex on it

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:8]: you don't need flex direction or justify content in this selector, just display flex

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking great, a few small updates are needed. Let me know if you have questions

File: css-layout-classes/styles.css (hunk 2 above)
Comment (saraysc) [reply css-layout-classes/styles.css]: image

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        <div class="column-one-fourth">
+          <img src="images/kirby.png" alt="kirby image">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" alt="fox image">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="pikachu image">
+        </div>
+        <div class="column-one-fourth">
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:38]: make sure to add this classname to the CSS with the proper flex basis on it

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <div id="container">
+      <h1>Choose your character!</h1>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:9]: this h1 should be wrapped in a `column-full` element with a flex-basis of 100%

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
almost there, a couple small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**